cdk/README.md

```diff
@@ -0,0 +1,46 @@
+# Kafka Connector for Amazon EventBridge Deployment on AWS
+
+This CDK stack is sample deployment to for the Kafka Connector for Amazon EventBridge. 
+
+> :warning: **This sample is not meant to be deployed to production environments!**
+
+## Overview
+
+This sample deploys several components in your AWS account:
+
+- Amazon MSK Serverless Cluster
+- Amazon S3 Bucket
+- IAM Roles & Security Groups
+- VPC & related components
+- Custom Amazon EventBridge Bus
+- Amazon ECS cluster with 2 Services (Data Producer & Transaction Analyzer)
+- (Optional) Amazon MSK Connector
+
+To learn more about those components please read the blog on the [AWS Big Data Blog](https://aws.amazon.com).
```

Comment:
Is there a better resource we can link ? This one seems too broad to be helpful.

Reply:
IMHO this is a placeholder for the final link

Reply:
Btw: the link should be highlighted in the top section of this README to showcase this is the accompanying resource of the official blog post.

Reply:
Yes this will link to the actual blog post we are publishing